modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -115,6 +126,55 @@ public static Map<String, Object> getActionVariables(
 		return variables;
 	}
 
+	public static Map<String, Object> getDDMValidationRuleVariables(
+			BaseModel<?> baseModel, DTOConverterRegistry dtoConverterRegistry,
+			ObjectDefinition objectDefinition,
+			ObjectEntryLocalService objectEntryLocalService,
+			JSONObject payloadJSONObject,
+			SystemObjectDefinitionMetadataRegistry
+				systemObjectDefinitionMetadataRegistry)
+		throws PortalException {
+
```

Comment:
@p-albuquerque why do we need a new method instead of reusing the `getValidationRuleVariables`?